### PR TITLE
Remove pancake

### PR DESCRIPTION
**Pull request by dominikwilkowski**
This proposal is a bit light on infos as I'm working through all the proposals. I may add more once I have had a deeper look into things

### CODE REVIEW

File: RFC-by-stage/1-approved/remove-pancake.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## The issue to be solved
+
+Pancake has been built (by me) a while ago and solved a couple very specific problems outlined in [this article](https://medium.com/dailyjs/npm-and-the-front-end-950c79fc22ce).
+Since 2018 the front end space has evolved significantly and shipping CSS files via npm isn't an issue anymore.
+In-fact shipping CSS files that are automatically imported by the prod bundle located inside a `node_modules/` folder causes significant issues to loaders like webpack which now require the CSS loader to be specified and the specifically allow the `node_modules/` path.
+
+## A short description of the solution
+
+We should remove pancake from the `postinstall` npm hook and deliver the CSS separately in the npm payload.
```
Comment (elisechant) [inline RFC-by-stage/1-approved/remove-pancake.md:16]: I read this that we will also still export Sass.

**Review by elisechant — APPROVED** [review]
Overwhelming yes.

At some stage this may cause package collisions. I see an alternative at that point to change repo from multi-package exports to single-package, but we might entertain that as a next step (another RFC).

File: RFC-by-stage/1-approved/remove-pancake.md (hunk 1 above)
Comment (dominikwilkowski) [reply RFC-by-stage/1-approved/remove-pancake.md:16]: No Sass will be exported I don't think. That's too hard as you need the dependency tree also installed and include it in the right order

Comment (elisechant) [reply RFC-by-stage/1-approved/remove-pancake.md:16]: OK. Can you confirm that you are saying this would remove Sass? if so, perhaps worth capturing.

**Comment by elisechant** [comment]
We might also wrap https://github.com/designsystemau/RFCs/pull/16 in to this work, given this: 

Screen Shot 2021-12-14 at 2 26 50 pm

File: RFC-by-stage/1-approved/remove-pancake.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The Sass files will then not be available anymore for integration as they currently feature lines like:
+
+```sass
+@import '../../../core/src/sass/_module.scss';
+```
+
+That path is only available within the monorepo.
+
+❗️ This means no more Sass exports.
```
Comment (elisechant) [inline RFC-by-stage/1-approved/remove-pancake.md:47]: This might alarm some, making a note to elevate this at Steering Committee.

Comment (dominikwilkowski) [reply RFC-by-stage/1-approved/remove-pancake.md:47]: I'm not sure anyone was using Sass from pancake. Worth asking

Comment (elisechant) [reply RFC-by-stage/1-approved/remove-pancake.md:47]: What if we do not remove Sass in this work, rather we distribute scss and css in lib (as opposed to dist):

/lib/buttons.scss  👈  `cp src/buttons.scss lib/buttons.scss`
/lib/buttons.css
/lib/button-react.jsx
/lib/button-js.js
/lib/button-jquery.jsx

where

/src/buttons.scss
etc

Comment (dominikwilkowski) [reply RFC-by-stage/1-approved/remove-pancake.md:47]: So I'm not proposing to remove Sass. I'm saying that removing pancake will effectively remove Sass from your import. 

The problem is that we have up to 4 Sass files in each component right now and some of them rely on the core sass that needs to be imported via a relative path. That relative path won't work in the node_modules environment.

**Pull request merged**